### DISCUSSION TITLE
Is it a way to pass DayPicker props to DayPickerInput?

### DISCUSSION THREAD

**Discussion by zieu** [gpbl/react-day-picker#1164] [Support] [answered]
As I am rendering only DayPickerInput I can't use some props available in DayPicker.

**Comment by gpbl** [comment, marked as answer]
Sure, use http://react-day-picker.js.org/api/DayPickerInput#dayPickerProps 

```jsx
<DayPicker dayPickerProps={{ ... }} />
```
